src/consts/models.tsx

```diff
@@ -12,6 +12,8 @@ export const CONST = {
   send: 'send',
   android: 'android',
   ios: 'ios',
+  transactionPassword: 'transactionPassword',
```

Comment:
maybe we could create a separate collection for this? Something like `const SECURE_STORAGE_KEYS = {
pin: 'pin' ...
}` ?

Reply:
I think it's just a way to messy consts.